UPGRADE.md

```diff
@@ -66,6 +66,46 @@
                 serialization_version: $version
                 serialization_groups: [Detailed]
  ```
+ 
+* Routing definition for Channels has been removed and replaced with auto generated resource routing. Channels are now resolved by code instead of id and only index and show endpoint are available.
```

Comment:
Would be nice to have some kind of an emphasis on the resources.
Also as for the structure, wouldn't it be more readable, if you've gone with something like:
```
* Routing for the following resources has been removed and replaced with the autogenerated routing:
- **channels** blah blah blah
- **countries** blah blah
- and so on blah blah
```
wdyt?